icons/hadoop/hadoop-original.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This one looks good to me, but it should be minimized. I recommend you check out our [Recommended resources and tools](https://github.com/devicons/devicon/wiki/Recommended-resources-and-tools). Specifically you should use SVGOMG for this, and follow the following guide: https://github.com/devicons/devicon/issues/1279#issuecomment-1195433256